src/fides/api/task/task_resources.py

```diff
@@ -166,3 +169,14 @@ def get_all_cached_erasures(self) -> Dict[str, int]:
         # extract request id to return a map of address:value
         number_of_leading_strings_to_exclude = 2
         return {extract_key_for_address(k, number_of_leading_strings_to_exclude): v for k, v in value_dict.items()}  # type: ignore
+
+    def get_connector(self, key: FidesKey) -> Any:
+        """Create or return the client corresponding to the given ConnectionConfig key"""
+        if key in self.connection_configs:
+            return self.connections.get_connector(self.connection_configs[key])
+        raise ConnectorNotFoundException(f"No available connector for {key}")
+
+    def close(self) -> None:
+        """Close any held resources"""
+        logger.debug("Closing all task resources for {}", self.request.id)
+        self.connections.close()
```

Comment:
I had also removed this originally when I removed DSR 2.0 altogether in the POC - this closes the open connections